src/db-api.ts

```diff
@@ -66,7 +76,19 @@ export async function connect() {
     process.exit();
   }
   port = +process.env.DB_PORT;
-  mysql_pool = mysql.createPool({
+  write_pool = mysql.createPool({
+    connectionLimit: 5,
+    connectTimeout: 30 * 1000,
```

Comment:
This is just a suggestion, but we have `Time` class now. With this you could write this like: `Time.seconds(30).toMillis()` which IMO makes things more easily readable.